proto/adaptors/frax/collateral_f_token.proto

```diff
@@ -0,0 +1,55 @@
+/*
+* Protos for function calls to the Frax Collateral F Token adaptor.
+*/
+
+syntax = "proto3";
+package steward.v3;
+
+option go_package = "/steward_proto";
+
+import "adaptors/base.proto";
+
+// Represents call data for the Frax Collateral F Token adaptor.
+message CollateralFTokenAdaptorV1 {
+    oneof function {
+        /***** BASE ADAPTOR FUNCTIONS *****/
+
+        // Represents function `revokeApproval(ERC20 asset, address spender)`
+        RevokeApproval revoke_approval = 1;
+
+        /***** ADAPTOR-SPECIFIC FUNCTIONS *****/
+
+        // Represents function `addCollateral(IFToken _fraxlendPair, uint256 _collateralToDeposit)`
+        AddCollateral add_collateral = 2;
+        // Represents function `removeCollateral(uint256 _collateralAmount, IFToken _fraxlendPair)`
```

Comment:
Maybe a dumb question but sanity checking: all the other Fraxlend functions start with the `IFToken`, is this an error or just an anomaly?

Reply:
they just changed the param name in the latest contracts, they are still `address` underneath

Reply:
I was referring to the order in the function call rather than the `IFToken` type.